src/ruby.ts

```diff
@@ -192,7 +192,8 @@ export class Ruby {
 
     if (major < 3) {
       throw new Error(
-        "The Ruby LSP requires Ruby 3.0 or newer to run. Please upgrade your Ruby version",
+        "The Ruby LSP requires Ruby 3.0 or newer to run. \
+        [Learn more](https://github.com/Shopify/vscode-ruby-lsp?tab=readme-ov-file#ruby-version-requirement)",
```

Comment:
```suggestion
        [Learn about using with older Ruby versions](https://github.com/Shopify/vscode-ruby-lsp?tab=readme-ov-file#ruby-version-requirement)",
```

Reply:
Do you think we need to be this explicit given that the message already mentions the Ruby version?

Reply:
My concern is that if people see "The Ruby LSP requires Ruby 3.0 or newer to run" then it sounds absolute, so why would they click to learn more.

Reply:
I changed to `See alternatives`.